### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gokulbnr**
Added a paper.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1555,6 +1555,8 @@ IEEE Robotics and Automation Letters, 7(3):8146-8153, 2022. [PDF](https://www.ir
 - <a name="Ji23arxiv"></a>Ji, X., Wei, J., Wang, Y., Shang, H., Kneip, L.,  
 *[Cross-modal Place Recognition in Image Databases using Event-based Sensors](https://arxiv.org/abs/2307.01047)*,  
 arXiv, 2023.
+*[How Many Events Do You Need? Event-Based Visual Place Recognition Using Sparse But Varying Pixels](https://ieeexplore.ieee.org/abstract/document/9925670)*,
```
Comment (guillermogb) [inline README.md:1558]: Using the DOI is preferable: https://doi.org/10.1109/LRA.2022.3216226

Comment (guillermogb) [reply README.md:1558]: Please look at the preview:
image

Authors are missing, and something is wrong with the formatting. This entry is mixing with the previous entry in the list ("Cross-modal place recognition"...). Please fix it.

Comment (gokulbnr) [reply README.md:1558]: Sorry about that, I'll take a quick look.

Comment (gokulbnr) [reply README.md]: Hopefully it's fixed now.

File: README.md [hunk 2]
```diff
@@ -1555,6 +1555,9 @@ IEEE Robotics and Automation Letters, 7(3):8146-8153, 2022. [PDF](https://www.ir
 - <a name="Ji23arxiv"></a>Ji, X., Wei, J., Wang, Y., Shang, H., Kneip, L.,  
 *[Cross-modal Place Recognition in Image Databases using Event-based Sensors](https://arxiv.org/abs/2307.01047)*,  
 arXiv, 2023.
+- <a name="Fischer22ral"></a>Fischer, T., Milford, M.,
+*[How Many Events Do You Need? Event-Based Visual Place Recognition Using Sparse But Varying Pixels](https://doi.org/10.1109/LRA.2022.3216226)*,  
+IEEE Robotics and Automation Letters (RA-L), 7(4):12275-12282, 2022. [PDF](https://arxiv.org/pdf/2206.13673), [Code](https://github.com/Tobias-Fischer/sparse-event-vpr)
```
Comment (guillermogb) [inline README.md:1560]: We're almost there...
image

Authors, paper title and publication venue in separate lines, please. You probably need to add two blank spaces after the comma in "M.," to signal a line break.

**Review by guillermogb — CHANGES_REQUESTED** [review]

**Review by guillermogb — APPROVED** [review]
thanks

**Pull request merged**